alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/DescriptorProcessor.java

```diff
@@ -60,16 +61,20 @@ public DescriptorProcessor(DescriptorMap descriptorMap, ConfigurationAccessor co
         for (AutoActionable autoActionable : autoActionables) {
             DistributionChannel channel = autoActionable.getChannel();
             ChannelKey channelKey = autoActionable.getChannelKey();
+            ConfigurationAction configurationAction = allConfigurationActions.get(channelKey.getUniversalKey());
```

Comment:
Does this work? I thought the point of `AutoActionable` was to create this for classes that don't declare it?

Reply:
Every channel will need a ConfigurationAction now as they all need to do the mapping for the JobDetailModels. However, not all of these ConfigurationActions need custom testing functionality. So this is still here to retrieve the existing ConfigurationActions and add a basic TestAction.